src/wallet.js

```diff
@@ -109,16 +109,16 @@ function isReceivable ({ status, walletType }: { status: Object, walletType: str
 }
 
 function isAddable ({ status, walletType }: { status: Object, walletType: string }) {
+  if (['drive', 'metamaskOne', 'referralWallet', 'escrow'].includes(walletType)) {
+    status.addable = false
+    status.disabledReason = status.disabledReason || 'Not addable'
```

Comment:
do not include disablereason for internal wallets

Reply:
our wallet disabled reason is not specific enough, causing some bugs such as when sending from coinbase wallet,
"not supported on mobile browser" is shown.

Suggest introducing [send|receive|add]DisabledReason

Reply:
I think it's better to include disabledReason for internal wallets, simple logic, whenever a wallet is disabled, state the reason.
Will introduce [send|receive|add]DisabledReason